pages/series/[slug].vue

```diff
@@ -35,19 +39,55 @@ onBeforeMount(async () => {
       <div class="content lg:px-8 pb-0">
         <div v-if="dataLoaded" class="grid">
           <div class="col">
-            <h1 class="mb-4 h2">{{ episode.title }}</h1>
+            <div class="series-logo-title mb-4">
+              <client-only>
+                <v-image-with-caption
```

Comment:
thank you :) totally missed the image on this

Reply:
:)